src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -51,6 +51,21 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
     @Param("visitsToDate") visitsToDate: LocalDate?,
   ): Int
 
+  @Query(
+    "select count(*) from visit v " +
+      " JOIN session_template st ON st.reference = v.session_template_reference " +
+      " WHERE st.reference = :reference" +
+      " AND v.visit_start > :visitsFromDate" +
+      " AND (cast(:visitsToDate as date) is null OR v.visit_start < :visitsToDate)" +
+      " AND visit_status = 'CANCELLED' AND (v.outcome_status is null OR v.outcome_status != 'SUPERSEDED_CANCELLATION') ",
```

Comment:
do migrated visits not have a outcome_status sometimes?

Reply:
Possible,... tested failed with out the v.outcome_status is null